src/components/ec-table-sort/ec-table-sort.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div
+    data-test="ec-table-sort"
+    class="ec-table-sort"
+  >
+    <div class="ec-table-sort__title">
+      <slot />
+    </div>
+    <a
+      class="ec-table-sort__icon-wrapper"
+      data-test="ec-table-sort__icon-wrapper"
+      href="#"
+      :title="directionTitle"
+      @click.prevent.stop="onSort"
+    >
+      <ec-icon
+        data-test="ec-table-sort__icon"
+        :size="20"
+        :name="icon"
+        :class="{
+          'ec-table-sort__icon': true,
+          'ec-table-sort__icon--asc': isAsc,
+          'ec-table-sort__icon--desc': isDesc
+        }"
+      />
+      <span class="ec-table-sort__text">{{ direction }}</span>
+    </a>
+  </div>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+import * as SortDirection from '../../enums/sort-direction';
+
+export default {
+  name: 'EcTableSort',
+  components: { EcIcon },
+  props: {
+    direction: {
+      type: String,
+      default: null,
+      validator(value) {
+        return value === null || value === '' || [SortDirection.ASC, SortDirection.DESC].includes(value);
+      },
+    },
+  },
+  computed: {
+    icon() {
+      if (this.isAsc || this.isDesc) {
+        return 'simple-arrow-drop-down';
+      }
+      return 'simple-arrow-up-down';
+    },
+    isAsc() {
+      return this.direction === SortDirection.ASC;
+    },
+    isDesc() {
+      return this.direction === SortDirection.DESC;
+    },
+    directionTitle() {
+      if (this.isAsc) {
+        return 'Sorted ascending';
```

Comment:
Would we want to add i18n in the future?

Reply:
Yeah, it's kind of complicated passing these 3 texts through table and table-head components. Now I regret adding this. I wanted to do it only for accessibility.

Reply:
let's keep it but I don't like these leftovers (shall we at least open a bug to keep track?)